index.js

```diff
@@ -1,12 +1,12 @@
 var fs = require("fs");
 
-var babelrc = fs.readFileSync("./.babelrc");
+var babelrc = fs.readFileSync("./babel.config.json");
```

Comment:
Hmm, perhaps this is operating on the target codebase.  The project we are using this with has a `babel.config.json` instead of a `.babelrc`.  Perhaps that is why this was necessary for us.  Perhaps we should try reading form either one?